release-notes/5.0/preview/api-diff/preview5/.Net/5.0-preview5_System.Runtime.Intrinsics.md

```diff
@@ -0,0 +1,16 @@
+# System.Runtime.Intrinsics
+
+``` diff
+ namespace System.Runtime.Intrinsics {
```

Comment:
System.Runtime.Intrinsics changes LGTM. CC. @CarolEidt, @echesakovMSFT 

The next preview (preview 7) should have all of ARM moved in box.